### PR TITLE
Feat: Add sidebar width methods to context provider

### PR DESCRIPTION
**Pull request by awcodes**
Adds `->sidebarWidth()` and `->collapsedSidebarWidth()` to context provider and Filament facade to allow setting sidebar widths without having to use a custom theme to do so.

### CODE REVIEW

File: packages/app/src/Context.php [hunk 1]
```diff
@@ -779,6 +783,30 @@ public function getSuccessColor(): array
         return $this->successColor ?? Color::Green;
     }
 
+    public function sidebarWidth(string $sidebarWidth): static
+    {
+        $this->sidebarWidth = $sidebarWidth;
+
+        return $this;
+    }
```
Comment (zepfietje) [inline packages/app/src/Context.php]:
```suggestion
    public function sidebarWidth(string $width): static
    {
        $this->sidebarWidth = $width;

        return $this;
    }
```

File: packages/app/src/Context.php [hunk 2]
```diff
@@ -779,6 +783,30 @@ public function getSuccessColor(): array
...
+    {
+        return $this->sidebarWidth;
+    }
+
+    public function collapsedSidebarWidth(string $collapsedSidebarWidth): static
+    {
+        $this->collapsedSidebarWidth = $collapsedSidebarWidth;
+
+        return $this;
+    }
```
Comment (zepfietje) [inline packages/app/src/Context.php]:
```suggestion
    public function collapsedSidebarWidth(string $width): static
    {
        $this->collapsedSidebarWidth = $width;

        return $this;
    }
```

File: packages/forms/src/Commands/MakeFormCommand.php [hunk 3]
```diff
@@ -41,7 +41,8 @@ public function handle(): int
             ->map(fn ($segment) => Str::lower(Str::kebab($segment)))
             ->implode('.');
 
-        $model = (string) str($this->argument('model') ?? ($this->option('edit') ?
+        $model = (string) str($this->argument('model') ?? (
+            $this->option('edit') ?
             $this->askRequired('Model (e.g. `Product`)', 'model') :
             $this->ask('(Optional) Model (e.g. `Product`)')
         ))->replace('/', '\\');
```
Comment (zepfietje) [inline packages/forms/src/Commands/MakeFormCommand.php:48]: This is quite unreadable to me now. 😅

**Review by zepfietje — CHANGES_REQUESTED** [review]

File: packages/forms/src/Commands/MakeFormCommand.php (hunk 3 above)
Comment (awcodes) [reply packages/forms/src/Commands/MakeFormCommand.php:48]: I didn't do anything to this file. Why is it in this?

Comment (awcodes) [reply packages/forms/src/Commands/MakeFormCommand.php:48]: Maybe this was added by github actions.

Comment (zepfietje) [reply packages/forms/src/Commands/MakeFormCommand.php:48]: Yeah seems like Pint. Weird it only picked up the changes in this PR. Let's leave it like this for now.

**Review by zepfietje — APPROVED** [review]
Code looks good to me.

**Comment by danharrin** [comment]
Thanks

**Pull request merged**